syntax = "proto3";

package protos;

import "protos/train.proto"
import "protos/eval.proto"
import "protos/input_reader"

message TrainEvalPipelineConfig {
  TrainConfig train_config = 1;
  EvalConfig eval_config = 2;
  InputReader train_input_reader = 3;
  InputReader eval_input_reader = 4;
}